.github/workflows/ci_test.yml

```diff
@@ -21,29 +21,67 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
+        # We're going to run the folowing to test basic functionality on all
+        # supported versions of Python:
+        #
+        #   pyNNN-test on ubuntu-latest for all supported versions of Python (3.6+)
```

Comment:
To avoid overusing resources, I would avoid py37, py38 and py39 -tests: we use py38 and py39 configurations for alldeps and other combinations, and I think we can assume that if 3.6 and all those work there is no need for the pyXX-test version

Reply:
Also, py310-test seems just a duplicate of py310-test-cov